### DISCUSSION TITLE
【求助】急急急！家属在空洞附近失踪了！

### DISCUSSION THREAD

**Discussion by OttoApocalypse01** [share121/inter-knot#3098] [其他]
![1eb5bacbaf00c21333120e2d602a4322](https://github.com/user-attachments/assets/b767e22f-903f-4ca5-b69d-accb78ea3908)
今天下午我可爱的乖孙女和我大吵一架后离家出走了，不久后在零号空洞附近失去信号，有任何蛛丝马迹请立刻致电天命科洛斯滕旧总部，提供有效线索者可以获得一万六千菲林的报酬，账记在北美支部上

**Comment by homteam** [comment]
这个委托我接了，不过现支付定金<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by I-m-Miss-Lucy-s-dog** [comment]
WC 一次保底 整層樓都別跟我搶

**Comment by seelesee** [comment]
我好像看见去月球找她侄女去了<img src="https://img-static.mihoyo.com/communityweb/upload/a70185899dfc90cdb13334ae8fdee289.gif" alt="乖巧" width="75" >
